File: BCCVL_Visualiser/bccvl_visualiser/scripts/test_xmlrpc.py
import xmlrpclib

# Create an object to represent our server.
server_url = 'http://localhost:6543/api.xml';
server = xmlrpclib.Server(server_url);

# Call the server and get our result.
result = server.xmlsss(5, 3)
print "content:", result['content']

File: BCCVL_Visualiser/bccvl_visualiser/scripts/test_xmlrpc.rb
require "xmlrpc/client"

# Make an object to represent the XML-RPC server.
server = XMLRPC::Client.new( "localhost", "/api.xml", 6543)

# Call the remote server and get our result
result = server.call("xml", "asds", "sadsd")

content = result["content"]

puts "content: #{content}"
